### DISCUSSION TITLE
fatal: home dir outside of repository

### DISCUSSION THREAD

**Discussion by abz123** [tpope/vim-fugitive#1958] [Q&A] [answered]
**part of .gitconfig**
[core]
	pager = 
	editor = nv
	excludesfile = .gitignore_global
	worktree = .

**in nvim, try to add files to index:**
:G add .

**error:**
fatal: home dir outside of repository

any suggestions please?

**Comment by tpope** [comment, marked as answer]
Don't set `worktree = .`. This will break lots of things, not just Fugitive.

**Reply by abz123** [reply]
Thanks, removed.
[core]
	pager = 
	editor = nv
	excludesfile = .gitignore_global
	worktree = 

Still get **error**:
:G add . 
Fatal: home dir outside repository?

**Reply by tpope** [reply]
Remove the `worktree` declaration entirely. This option is meant to be set on an individual repository, not globally, and to an absolute path.

**Reply by abz123** [reply]
worktree completely removed:
[core]
	pager = 
	editor = nv
	excludesfile = .gitignore_global

previous error persists:
:G add .
Fatal: home dir outside repository?

**Reply by tpope** [reply]
What is the **exact** error message? The first time you said `fatal: home dir outside of repository` but now you're saying `Fatal: home dir outside repository?`.
